src/site/content/en/lighthouse-accessibility/audio-caption/index.md

```diff
@@ -30,10 +28,26 @@ See glitch: [meggin-accessibility-assets](https://glitch.com/edit/#!/meggin-acce
 
 {% include 'content/lighthouse-accessibility/scoring.njk' %}
 
-## How to add an audio track
+## How to add captions to an audio clip
 
-Add at least one track element to the `audio` element
-with attribute `kind="captions"`:
+For every `<audio>` element on your page, add at least one `<track>` element
+with the attribute `kind="captions"`:
+
+```html
+<audio>
+   <source src="audioSample.mp3" type="audio/mp3">
+   <track src="captions_en.vtt" kind="captions" srclang="en" label="captions">
+</audio>
+```
+
+The source for the track must be a text file in the
+[WebVTT format](https://developer.mozilla.org/en-US/docs/Web/API/WebVTT_API).
+Captions should include all essential information about the audio clip,
```

Comment:
Does this duplicate the information in the why-captions include on line 13?

Reply:
Yes, there's a little duplication here, but that occurs not infrequently in the audit posts. I actually think it's probably not a bad practice to compensate for selective reading. For example, my hunch is a fair number of users will skip to the "How to fix it" section, so having similar info there and in the "what are captions for anyway" section is probably doing some good. But I'm OK with omitting if you feel strongly.

Reply:
not a strong opinion. fine with leaving it.